certification/internal/shell/has_prohibited_packages.go

```diff
@@ -62,7 +62,7 @@ func (p *HasNoProhibitedPackagesCheck) Name() string {
 }
 func (p *HasNoProhibitedPackagesCheck) Metadata() certification.Metadata {
 	return certification.Metadata{
-		Description:      "Checks to ensure that the image in use does not contain prohibited packages.",
+		Description:      "Checks to ensure that the image in use does not include Red Hat Enterprise Linux (RHEL) kernel packages.",
```

Comment:
I _think_ the packages that we don't want to see include more than just kernel-related packages. Can you instead say something like `does not include prohibited packages, such as ...`